From b685228ca0540a8b8f682f065c53d65e1c4c0711 Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Sat, 30 Sep 2023 18:22:53 +0200
Subject: [PATCH 501/954] feat(stm32mp21): remove GPIOJ and GPIOK

On STM32MP21, there are no GPIOJ and GPIOK instance.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git cdc4470764fb4c53e69d4235737edbed20670223 feat(stm32mp21): remove GPIOJ and GPIOK]
Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: I53d40691e4d084166c5d13c6475a6e821c8d9256
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/356320
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp2/stm32mp2_def.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index 8c9c5c70da..c2d4da07c0 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -325,8 +325,10 @@ enum ddr_type {
 #define GPIOG_BASE				U(0x442A0000)
 #define GPIOH_BASE				U(0x442B0000)
 #define GPIOI_BASE				U(0x442C0000)
+#if !STM32MP21
 #define GPIOJ_BASE				U(0x442D0000)
 #define GPIOK_BASE				U(0x442E0000)
+#endif /* !STM32MP21 */
 #define GPIOZ_BASE				U(0x46200000)
 #define GPIO_BANK_OFFSET			U(0x10000)
 
-- 
2.53.0

